components/PostList.tsx

```diff
@@ -0,0 +1,55 @@
+import Link from "next/link";
+
+export default function PostList({ posts, admin }) {
```

Comment:
change all functions syntax -> functional components (see image for example) 
<img width="275" alt="image" src="https://user-images.githubusercontent.com/59902670/190164552-bcde25fb-b72f-4833-a58e-3ba528f9c2b6.png">